recipe/conda_build_config.yaml

```diff
@@ -43,6 +43,10 @@ m2w64_fortran_compiler:        # [win]
 
 cuda_compiler:                 # [linux or win]
   - nvcc                       # [linux or win]
+  - nvcc                       # [linux or win]
```

Comment:
```suggestion
  - nvcc                       # [linux64 or win]
```

Reply:
Gotcha. Changed and added to the CUDA arch migrator